dtfs-central-api/src/v1/controllers/utilisation-report-service/get-fee-record.controller/index.test.ts

```diff
@@ -2,19 +2,39 @@ import httpMocks from 'node-mocks-http';
 import { HttpStatusCode } from 'axios';
 import { when } from 'jest-when';
 import { FeeRecordEntityMockBuilder, UtilisationReportEntityMockBuilder } from '@ukef/dtfs2-common';
-import { getFeeRecordDetails, GetFeeRecordDetailsResponseBody } from '.';
-import { mapFeeRecordEntityToDetails } from './helpers';
+import { getFeeRecord, GetFeeRecordResponseBody } from '.';
+import { mapFeeRecordEntityToResponse } from './helpers';
 import { FeeRecordRepo } from '../../../../repositories/fee-record-repo';
 
 console.error = jest.fn();
 
 jest.mock('./helpers');
 
-describe('get-fee-record-details.controller', () => {
-  describe('getFeeRecordDetails', () => {
+describe('get-fee-record.controller', () => {
+  describe('getFeeRecord', () => {
     const reportId = 3;
     const feeRecordId = 14;
 
+    const aFeeRecordResponseBody = (): GetFeeRecordResponseBody => ({
+      id: 123,
+      bank: {
+        id: '456',
+        name: 'A bank',
+      },
+      reportPeriod: {
+        start: {
+          month: 1,
+          year: 2024,
+        },
+        end: {
+          month: 1,
+          year: 2024,
+        },
+      },
```

Comment:
We have an `aReportPeriod` helper defined already which we can call to populate this, and we could also pull out a `aSessionBank` helper too to keep this nice and short.